### PR TITLE
Added Hit logic to move pawn to inner circle

### PR DESCRIPTION
**Pull request by mohithsrao**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,7 @@ export.cfg
 .mono/
 
 #Build and Release ignores
-Bin/
\ No newline at end of file
+Bin/
+
+#VSCode-specific ignores
+.vscode
```
Comment (nofea) [inline .gitignore]: .vscode/ , add "/"

File: ChoukaBaraGame/Scripts/Game/GameUtility.gd [hunk 2]
```diff
@@ -2,6 +2,8 @@ extends Node
 
 var goalPosition : Vector2 = Vector2(480,480)
 var Group_Player = "gp_player"
+var distanceToGoal:int = 24
```
Comment (nofea) [inline ChoukaBaraGame/Scripts/Game/GameUtility.gd:5]: Where are these used?

File: ChoukaBaraGame/Scripts/Player/Player.gd [hunk 3]
```diff
@@ -15,6 +15,7 @@ onready var homeBase = $HomeBase
 var pawnScene = preload("res://Scenes/Player/Pawn.tscn")
 var maxCharactersPerPlayer = 4
 var needsReRoll:bool = false
+var canEnterInnerCircle = false
```
Comment (nofea) [inline ChoukaBaraGame/Scripts/Player/Player.gd:18]: What is this used for?

File: ChoukaBaraGame/project.godot [hunk 4]
```diff
@@ -68,6 +68,7 @@ GameUtility="*res://Scripts/Game/GameUtility.gd"
 [debug]
 
 gdscript/completion/autocomplete_setters_and_getters=true
+gdscript/warnings/unused_class_variable=true
```
Comment (nofea) [inline ChoukaBaraGame/project.godot:71]: Is this tell me that there are used class variables in your code???

**Review by nofea — COMMENTED** [review]
Revert with justifications

File: ChoukaBaraGame/Scripts/Game/GameUtility.gd (hunk 2 above)
Comment (mohithsrao) [reply ChoukaBaraGame/Scripts/Game/GameUtility.gd:5]: This is required for the next requirement for Goal Implementation.
Currently not getting used.

File: ChoukaBaraGame/Scripts/Player/Player.gd (hunk 3 above)
Comment (mohithsrao) [reply ChoukaBaraGame/Scripts/Player/Player.gd:18]: Pawn can enter inner circle only when the player has a hit under his belt. other wise he has to move another pawn.
This is the flag at player level to track that.

File: ChoukaBaraGame/project.godot (hunk 4 above)
Comment (mohithsrao) [reply ChoukaBaraGame/project.godot:71]: This is part of the Lint to keep clean code. it's for everyone.

Comment (nofea) [reply ChoukaBaraGame/project.godot:71]: So I assume this has already been addressed in the previous comments?

**Review by nofea — APPROVED** [review]
Approved!!!

**Pull request merged**